rfd/0196-introducing-a-component-system.md

```diff
@@ -0,0 +1,518 @@
+---
+authors: Ryan Clark (ryan.clark@goteleport.com)
+state: draft
+---
+
+# RFD 0194 - Introducing a component system
+
+## Required Approvers
+
+* Engineering: @ravicious || @kimlisa || @avatus
+* Design: @roraback
+
+## What
+
+Introduce Chakra UI, a 3rd-party component system with pre-built components and easy styling capabilities into our web
+UI.
+
+## Why
+
+Modernise our ability to create styled, reusable, useful components without having to reinvent them from scratch each
+time. Speed up development time by having pre-built, tested components ready to use with minimal fuss. Accessibility,
+including ARIA support and keyboard navigation, is built in to all components.
```

Comment:
These are big points IMO. Using pre-tested base components could allow us to ditch a lot of existing frontend smoke-tests, and having solid baked-in a11y would be great for more consistent keyboard nav.